tests/bootstrap.php

```diff
@@ -159,7 +159,7 @@
 Settings::Set('system.documentroot_use_default_value', '1', true);
 Settings::Set('system.hostname', 'dev.froxlor.org', true);
 Settings::Set('system.nameservers', 'dev.froxlor.org', true);
-Settings::Set('system.mysql_access_host', 'localhost,127.0.0.1,2a01:440:1:12:82:149:225:46,82.149.225.46', true);
+Settings::Set('system.mysql_access_host', 'localhost,127.0.0.1,172.17.0.1,2a01:440:1:12:82:149:225:46,82.149.225.46', true);
```

Comment:
what's `172.17.0.1` doing in there?

Reply:
Allow the docker host to access the docker instance.

Reply:
but you're using `127.0.0.1` in all connections, just don't see the need for this then :) it's more or less just for customer database users - but default setting for most mysql/mariadb servers is they are bound to localhost/127.0.0.1 anyway :)

Reply:
> but you're using `127.0.0.1` in all connections, just don't see the need for this then :) it's more or less just for customer database users - but default setting for most mysql/mariadb servers is they are bound to localhost/127.0.0.1 anyway :)

I am talking to `127.0.0.1:3306` which forwards packets to `172.17.0.2:3306` where docker is running on.
This option is not what the DST host can be, but what the SRC host can be. And when accessing `172.17.0.2:3306` (DST) the SRC is `172.17.0.1`, so this needs to be added to the allowed hosts.

Reply:
as long as it works :P it's for testing only anyway :) I never got too deep into this whole docker thing - thanks for your work, it's really appreciated

Reply:
I had to get into docker for work and I like it even less than before, but in this case (travis-ci matrix) its unfortunately the only way to test it with different database servers :/

Reply:
BTW so you can understand why I really had to add the IP `172.17.0.1`: https://travis-ci.com/Froxlor/Froxlor/builds/120991797
This job failed because of:
```
  [phpunit] There was 1 failure:
  [phpunit] 
  [phpunit] 1) MysqlsTest::testCustomerMysqlsAdd
  [phpunit] SQLSTATE[HY000] [1045] Access denied for user 'test1sql1'@'172.17.0.1' (using password: YES)
  [phpunit] 
  [phpunit] /home/travis/build/Froxlor/Froxlor/tests/Mysqls/MysqlsTest.php:45
```

Reply:
> I had to get into docker for work and I like it even less than before, but in this case (travis-ci matrix) its unfortunately the only way to test it with different database servers :/

but definetly a very nice feature in this case and really helpful, thanks for that again